backend/core/src/listings/listings.service.ts

```diff
@@ -49,62 +49,77 @@ export class ListingsService {
   }
 
   public async list(origin: string, params: ListingsQueryParams): Promise<PaginatedListingsDto> {
-    let qb = this.getQueryBuilder()
+    // Inner query to get the sorted listing ids of the listings to display
+    // TODO(avaleske): Only join the tables we need for the filters that are applied
+    const innerFilteredQuery = this.listingRepository
+      .createQueryBuilder("listings")
+      .select("listings.id", "listings_id")
+      .leftJoin("listings.property", "property")
+      .leftJoin("property.units", "units")
```

Comment:
Do we need to join the units table? IIUC we're constrained to only be able to filter on listing.status, listing.name, and property.neighborhood (https://github.com/CityOfDetroit/bloom/blob/4093e1e9b055ed514deb5152f14651b213eda124/backend/core/src/listings/dto/listing.dto.ts#L767-L771). And if we're grouping by listings.id, are we getting anything from the units table?

Reply:
For now we don't need it, so removing. Once we filter on unit types and such, we'll probably need to add it back